### PR TITLE
FireEyeHX Create Indicator Enhacement

### PR DESCRIPTION
**Pull request by mohlcyber**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Updated the ***fireeye-hx-create-indicator*** command to support additional optional arguments. 
This includes - display_name, description and platforms

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml [hunk 1]
```diff
@@ -1873,6 +1873,29 @@ script:
       default: false
       isArray: false
       secret: false
+    - description: Display Name for the Indicator.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml:1876]:
```suggestion
    - description: Display Name for the indicator.
```

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml [hunk 2]
```diff
@@ -1873,6 +1873,29 @@ script:
...
+      name: display_name
+      required: false
+      default: false
+      isArray: false
+      secret: false
+    - description: Description for the Indicator.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml:1882]:
```suggestion
    - description: Description for the indicator.
```

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml [hunk 3]
```diff
@@ -1873,6 +1873,29 @@ script:
...
+      name: description
+      required: false
+      default: false
+      isArray: false
+      secret: false
+    - auto: PREDEFINED
+      description: Select Platform for the Indicator. If not selected, Indicator will be created for all platforms.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.yml:1889]:
```suggestion
      description: The platform for the indicator. If not selected, the indicator will be created for all platforms.
```

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md [hunk 4]
```diff
@@ -2107,8 +2107,11 @@ Create a new indicator.
 #### Input
 
 | **Argument Name** | **Description** | **Required** |
-| --- | --- | --- |
-| category | The indicator category. | Required | 
+| --- | --- |----------|
+| category | The indicator category. | Required |
+| display_name | Display Name for the Indicator. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md:2112]:
```suggestion
| display_name | Display name for the indicator. | Optional |
```

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md [hunk 5]
```diff
@@ -2107,8 +2107,11 @@ Create a new indicator.
...
+| description | Description for the Indicator. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md:2113]:
```suggestion
| description | Description for the indicator. | Optional |
```

File: Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md [hunk 6]
```diff
@@ -2107,8 +2107,11 @@ Create a new indicator.
...
+| platforms | Select Platform for the Indicator. If not selected, Indicator will be created for all platforms. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/Integrations/FireEyeHXv2/README.md:2114]:
```suggestion
| platforms | The platform for the Indicator. If not selected, the indicator will be created for all platforms. | Optional |
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**